core/src/main/java/org/mobilitydata/gtfsvalidator/annotation/GtfsEnumValue.java

```diff
@@ -0,0 +1,12 @@
+package org.mobilitydata.gtfsvalidator.annotation;
+
+import java.lang.annotation.*;
+
+@Target(ElementType.TYPE)
+@Retention(RetentionPolicy.SOURCE)
+@Repeatable(GtfsEnumValues.class)
+public @interface GtfsEnumValue {
```

Comment:
/**
 \* Annotation to specify an enum name-value pair. The pair will be used to generate an actual Enum that will be used by the processing code.
 */

Reply:
Done.